### DISCUSSION TITLE
Does this library provide any function for changing the duration of a PLC timeout?

### DISCUSSION THREAD

**Discussion by ctsaitsao** [FreeOpcUa/python-opcua#1392] [Q&A] [answered]
I am sending a large amounts of requests to the PLC and have a retry mechanism in which I resend the requests if a `TimeoutError` occurs. I would like to decrease the amount of time that elapses before a `TimeoutError` is raised in order to expedite the retry process.

**Comment by AndreasHeine** [comment, marked as answer]
https://github.com/FreeOpcUa/python-opcua/blob/3012acf9047aad3acc2674e9008f0d1eee680c25/opcua/client/client.py#L95

but respect ServerCapabilities and OperationalLimits!
